openff/interchange/_tests/unit_tests/interop/lammps/export/test_lammps.py

```diff
@@ -62,3 +68,104 @@ def test_to_lammps_single_mols(self, mol, sage_unconstrained, n_mols):
                 "Torsion": 0.02 * unit.kilojoule_per_mole,
             },
         )
+
+    def test_unique_lammps_mol_ids(
+        self, smiles: str, sage_unconstrained: ForceField, sidelen: int = 0
```

Comment:
```suggestion
        self, sage_unconstrained: ForceField,
```
If `sidelen` is only one value, might as well leave it out of the (test) function definition. It'd make sense to include as an argument into some positions-building helper function if the middle of this test is refactored out.

I think you might also might be getting tripped up about how arguments are passed to `pytest`-style tests in general; most tests either don't need any inputs (save `self` with this class-based organization I like) or fixtures (which you can think of as just constants). Arguments that might take different values only make sense if they're being managed by the fancy magic of a `pytest.mark.parametrize` decorator, so `smiles` here isn't even defined, I don't think. If you're just looking to test one molecule, I'd drop this argument and just directly call `Molecule.from_smiles("CCO")` or whatever down below.

Reply:
I had superficially based the structure of the inputs to my test on the [parameters for `test_to_lammps_single_mols()` above](https://github.com/openforcefield/openff-interchange/blob/lammps-hotfix/openff/interchange/_tests/unit_tests/interop/lammps/export/test_lammps.py#L16-L26) without having a great sense of how these parameters interact, thanks for clarifying this to me.

`sidelen` can be any arbitrary positive int (except 1, since you can't test for uniqueness when there's only 1 molecule present) and I left it as a free parameter accordingly; incidentally, I now notice the default value in this commit of `sidelen=0` is wrong for this reason. Would it be better to hard-code an arbitrary value within the function block (e.g. 3) or pass in values via `pytest.mark.parametrize`? Same goes for `smiles`, I wanted to be able to test any molecule to make there's nothing chemistry-specific to the test (namely for water or other "special" molecules). Should I hard-code a smiles string or just parameterize as with `test_to_lammps_single_mols()`?

Reply:
I've parameterized these fields using `pytest.mark.parameterize()` in [this commit](https://github.com/openforcefield/openff-interchange/commit/652ad4359a48bd03aed9f9a06bbdfb05cfc5a25a), but I'd be more than happy to nix these changes if they're errant.

Reply:
`parameterize` is a good solution here